### ISSUE TITLE
Issue with Code Reproduction

### ISSUE THREAD

**Issue by syt06007** [kristian-fer/ByteRCNN#2]
Hello, First of all, thank you for providing the open-source code. 

I am encountering an issue while running the Tensorflow code. When running Scenario 1 with a size of 512, 
I noticed that both the train accuracy and validation accuracy start to decline around the 3rd epoch. 

Is there another version of the code implemented that I could try? If not, how can I resolve this issue? There have been no changes made to the code during execution.
![image](https://github.com/kristian-fer/ByteRCNN/assets/70042492/2bdc5ca4-22d6-4940-9ec1-30d8906c68cd)

**Comment by kristian-fer** [comment]
Thank you for your interest in this work. Please try adapting the learning rate used during training.
The issue you described points to that. 
We had positive results with LR = 0.0001 and reported them in the paper. 
We encourage you to do further experiments on parameter fine tuning as well.
